group_vars/plantain/plantain_staging.yml

```diff
@@ -0,0 +1,50 @@
+---
+postgres_host: 'lib-postgres3.princeton.edu'
+postgres_version: 10
+postgresql_is_local: false
+passenger_server_name: "pulfa3-staging1.princeton.edu"
+passenger_app_root: "/opt/plantain/current/public"
+passenger_app_env: "staging"
+nodejs_version: "9.x"
+bundler_version: "2.0.2"
```

Comment:
@jrgriffiniii pointing this out

Reply:
Did you happen to test this with a `capistrano` deployment?  If so, then I can merge https://github.com/pulibrary/plantain/pull/25

Reply:
I've just found this on the staging server, I shall update the documentation for `pulibrary.ruby`.